Gemfile.lock

```diff
@@ -55,52 +55,52 @@ GEM
       ransack (~> 2.1, >= 2.1.1)
       sassc-rails (~> 2.1)
       sprockets (>= 3.0, < 4.1)
-    activejob (6.0.2.2)
-      activesupport (= 6.0.2.2)
+    activejob (6.0.3.1)
+      activesupport (= 6.0.3.1)
       globalid (>= 0.3.6)
-    activemodel (6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activerecord (6.0.2.2)
-      activemodel (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activestorage (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
+    activemodel (6.0.3.1)
+      activesupport (= 6.0.3.1)
+    activerecord (6.0.3.1)
+      activemodel (= 6.0.3.1)
+      activesupport (= 6.0.3.1)
+    activestorage (6.0.3.1)
+      actionpack (= 6.0.3.1)
+      activejob (= 6.0.3.1)
+      activerecord (= 6.0.3.1)
       marcel (~> 0.3.1)
-    activesupport (6.0.2.2)
+    activesupport (6.0.3.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.2)
+      zeitwerk (~> 2.2, >= 2.2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
     airbrussh (1.4.0)
       sshkit (>= 1.6.1, != 1.7.0)
     annotate (3.1.1)
       activerecord (>= 3.2, < 7.0)
       rake (>= 10.4, < 14.0)
-    apipie-rails (0.5.17)
+    apipie-rails (0.5.18)
       rails (>= 4.1)
     arbre (1.2.1)
       activesupport (>= 3.0.0)
     ast (2.4.0)
-    aws-eventstream (1.0.3)
-    aws-partitions (1.289.0)
-    aws-sdk-core (3.92.0)
-      aws-eventstream (~> 1.0, >= 1.0.2)
+    aws-eventstream (1.1.0)
```

Comment:
### [aws-eventstream](https://github.com/aws/aws-sdk-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.0.3 → 1.1.0

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-eventstream/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-eventstream)_]





<details>
<summary>Commits</summary>

A change of **3** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/1.0.3...1.1.0).

These are the individual commits:
- (b39e2a0) [Version 1.0.4 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/b39e2a0478c460753e535c990be1f60fb5d8df9f)
- (db68e48) [Correct documentation. Example should show method call filte…](https://github.com/aws/aws-sdk-ruby/commit/db68e484d5653c8b26e11d11620d52280e0d5ab8)
- (0d31588) [Version 1.1.0 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/0d31588d6e80cfc0ec9bd53d48273c1b609fad8f)


</details>